Sources/AWSLogging/CloudwatchJsonStandardErrorLoggerV2.swift

```diff
@@ -145,11 +154,14 @@ public struct CloudwatchJsonStandardErrorLoggerV2: LogHandler {
             shortFileName = file
         }
         
-        let metadataToUse: Logger.Metadata
+        var metadataToUse: Logger.Metadata = self.metadata
+                
+        if let provided = self.metadataProvider?.get() {
+            metadataToUse = metadataToUse.merging(provided) { (global, local) in local }
```

Comment:
You can probably just merge in place (i.e. `.merge` instead of `.merning`)

Reply:
Yes, fixed.